test/version-test.js

```diff
@@ -1,13 +1,14 @@
+import { version } from '../package.json';
```

Comment:
Considering 6198988c7729f21268a1db6c9bf03463cb342dda, maybe we should undo this line?

Reply:
Yeah I’d prefer if this tested the version in the UMD module, rather than reading the version independently.

Reply:
5013079 - test passing